app/views/content/life-as-a-teacher/teaching-as-a-career/from-teaching-intern-to-leadership-roles.md

```diff
@@ -0,0 +1,18 @@
+---
+title: From teaching intern to leadership roles
```

Comment:
```suggestion
title: From teaching internship to leadership roles
```
would suggest using `internship` rather than `intern` as this is much more prevalent in terms of keywords in GSC?